its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/ScannerMSBuildTest.java

```diff
@@ -760,64 +727,40 @@ void testCSharpFramework48() throws IOException {
   }
 
   @Test
-  void testCSharpSdk2() throws IOException {
-    validateCSharpSdk("CSharp.SDK.2.1");
-  }
-
-  @Test
-  void testCSharpSdk3() throws IOException {
-    validateCSharpSdk("CSharp.SDK.3.1");
+  void testCSharpSdk6() throws IOException {
+    validateCSharpSdk("CSharp.SDK.6");
   }
 
   @Test
-  void testScannerNetCore31NoAnalysisWarning() throws IOException {
-    assumeFalse(TestUtils.getMsBuildPath(ORCHESTRATOR).toString().contains("2017")); // We can't run .NET Core SDK under VS 2017 CI context
-    Path projectDir = TestUtils.projectDir(basePath, "CSharp.SDK.3.1");
-    BuildResult buildResult = runNetCoreBeginBuildAndEnd(projectDir, ScannerClassifier.NET);
-
-    assertThat(buildResult.getLogs()).doesNotContain("Failed to parse properties from the environment variable 'SONARQUBE_SCANNER_PARAMS'");
-    assertUIWarnings(buildResult);
-  }
-
-  @Test
-  void testCSharpSdk5() throws IOException {
-    validateCSharpSdk("CSharp.SDK.5");
-  }
+  void testScannerNet6NoAnalysisWarnings() throws IOException {
+    // dotnet sdk tests should run only on VS 2022
+    assumeTrue(TestUtils.getMsBuildPath(ORCHESTRATOR).toString().contains("2022"));
```

Comment:
Is this still needed? What happens without it?

Reply:
Yes, still needed, otherwise we get:
```
C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\MSBuild\15.0\Bin\Microsoft.Common.CurrentVersion.targets(1179,5): error MSB3644: The reference assemblies for framework ".NETFramework,Version=v6.0" were not found. To resolve this, install the SDK or Targeting Pack for this framework version or retarget your application to a version of the framework for which you have the SDK or Targeting Pack installed. Note that assemblies will be resolved from the Global Assembly Cache (GAC) and will be used in place of reference assemblies. Therefore your assembly may not be correctly targeted for the framework you intend.
```